TranslationBackend/app/Http/Controllers/WordController.php

```diff
@@ -44,15 +44,31 @@ public function show()
         if ( request()->has( 'sequence' ) ) {
             $sequence = request()->sequence;
             $word = null;
+
             if ( request()->has( 'word' ) ) {
                 $word = request()->word;
             }
+
+            // default page number is 1
+            $pageNum = 1;
+            if(request() -> has( 'pageNum')) {
+                $pageNum = request()->pageNum;
+            }
+
+            // default rows in a page is 10
+            $pageRows = 10;
+            if(request() -> has( 'pageRows')) {
```

Comment:
Yea idk how i feel about the pageRows being in the endpoint, it doesn't seem right to put it there.....